cosmic/sample/sampler/independent.py

```diff
@@ -208,8 +208,8 @@ def sample_primary(self, primary_model='kroupa01', size=None):
             multiplier = 1
             a_0 = np.random.uniform(0.0, 1, size)
 
-            low_cutoff = 0.740074
-            high_cutoff = 0.908422
+            low_cutoff = 0.771
+            high_cutoff = 0.919
```

Comment:
If you have a broken power law, you need to pick the relative contribution of each of them. The cutoffs set the normalization of what fraction of stars are distributed according to each power law.  The numbers come from Hurley+2002 Eqs 95-96. You get a similar cutoff for kroupa01. 

You can also verify that this produces the right behavior because the histograms don't have big jumps at the masses where the power law switches:

![primary_mass_kroupa93](https://user-images.githubusercontent.com/22921809/80592473-82076a80-89ed-11ea-9ed3-fd42f38fcc75.png)
![primary_mass_kroupa_01](https://user-images.githubusercontent.com/22921809/80592482-85025b00-89ed-11ea-91f5-d4f10c3ca128.png)

Reply:
the top one is kroupa93 and the bottom one is kroupa01. The differences are small, but verifiable by the tests.

Reply:
ok...why don't the  cutoffs normalize to unity then? Sorry for not being used to how broken power laws are generated :)

Reply:
nevermind, I  see that the distribution integrates to unity